docs/tex/api/reference.tex

```diff
@@ -0,0 +1,93 @@
+\title{Reference}
+
+{{navbar}}
+
+\subsubsection{Models}
+
+{{sphinx
+
+* :class:`edward.models.RandomVariable`
+  
+  * :class:`edward.models.Bernoulli`
```

Comment:
Is there any way we can link these subclasses to TensorFlow distribution's API? Not a big priority, we can probably hack it when post-processing Sphinx blocks. I only wonder if Sphinx has a built-in solution.